src/components/cards/TradeCardGP/TradeCardGP.vue

```diff
@@ -53,20 +53,34 @@
         block
         @click.prevent="handlePreviewButton"
       />
-      <div
-        class="mt-6 bg-gray-50 rounded text-sm p-3 grid gap-2 grid-flow-col text-gray-600 dark:bg-gray-800 dark:text-gray-400"
-        v-if="trading.isBalancerTrade.value"
-      >
-        <LightBulbIcon />
-        <span>
-          {{ $t('tradesThroughWeth') }}
-          <a
-            @click="switchToWETH()"
-            class="text-blue-500"
-            v-text="$t('tradeFromWeth')"
-          />
-          {{ $t('saveGas') }}
-        </span>
+      <div class="mt-6 text-sm flex items-center">
+        <BalTooltip
+          width="64"
+          :disabled="!trading.isGaslessTradingDisabled.value"
```

Comment:
`.value` shouldn't be required in templates. What's going on here?

Reply:
hmm something doesn't seem right here. Could you wrap the `trading` export in `toRefs(trading)` to see if that helps?